docs/src/main/asciidoc/spring-cloud-stream-support.adoc

```diff
@@ -892,6 +1007,109 @@ See the <<si-sb-headers, Service Bus message headers>> for the basic message hea
 
 NOTE: When setting the partiton key, the priority of message header is higher than Spring Cloud Stream property. So `spring.cloud.stream.bindings.<binding-name>.producer.partition-key-expression` will take effect only when none of the headers of `ServiceBusMessageHeaders#SESSION_ID`, `ServiceBusMessageHeaders#PARTITION_KEY`, `AzureHeaders#PARTITION_KEY` is configured.
 
+===== Multi Binders Support
+Step 1. To use multiBinders of ServiceBus, we need to configure below properties in application.yml
+[source,yaml]
+----
+spring:
+  cloud:
+    stream:
+      function:
+        definition: consume1;supply1;consume2;supply2
+      bindings:
+        consume1-in-0:
+          destination: ${SERVICEBUS_TOPIC_NAME}
+          group: ${SUBSCRIPTION_NAME}
+        supply1-out-0:
+          destination: ${SERVICEBUS_TOPIC_NAME_SAME_AS_ABOVE}
+        consume2-in-0:
+          binder: servicebus-2
+          destination: ${SERVICEBUS_QUEUE_NAME}
+        supply2-out-0:
+          binder: servicebus-2
+          destination: ${SERVICEBUS_QUEUE_NAME_SAME_AS_ABOVE}
+      binders:
+        servicebus-1:
+          type: servicebus
+          default-candidate: true
+          environment:
+            spring:
+              cloud:
+                azure:
+                  servicebus:
+                    namespace: ${SERVICEBUS_NAMESPACE_01}
```

Comment:
same here, should be connection-string
![image](https://user-images.githubusercontent.com/63028776/154920872-40602196-5e22-433f-ac7a-f4951bbcd965.png)